automation/Commands/profile_commands.py

```diff
@@ -62,7 +61,8 @@ def save_flash_files(logger, browser_params, dump_location, clear=False):
 
         if clear:
             logger.debug("BROWSER %i: CLEARING %s during flash file archive" % (browser_params['crawl_id'], location))
-            rmsubtree(location)
+            shutil.rmtree(location)
```

Comment:
I think this change might break things, but I'm not completely sure. If I remember correctly, the reason for adding `rmsubtree` is that `shutil.rmtree` deletes `location` (the parent directory, `#SharedObjects`), so instead we spider within the directory and delete all files and subdirectories.

Did you confirm that flash will recreate the `#SharedObjects` directory after deleting it?